webpack.config.js

```diff
@@ -1,17 +1,9 @@
 const path = require('path');
-const webpackConfig = require('terra-dev-site/src/config/webpack.config');
-const aggregateTranslations = require('./packages/terra-i18n/scripts/aggregate-translations/aggregate-translations');
-
-aggregateTranslations({ baseDirectory: __dirname });
+const webpackConfig = require('terra-dev-site/src/webpack/webpack.config')();
```

Comment:
From looking how we import this and other sub components like out of terra-form and the tabs, would it be possible to get terra-dev-site building out to a lib folder instead of an src one?

Reply:
I'd like to leave sorting that out to another PR. There isn't a reason to transpile the webpack config because it's executed by node. But we do have some code in terra-dev-site intended for the client side and I think we should transpile that.

Reply:
I do like that approach.

Reply:
I'll make that change in terra-dev-site and terra-toolkit